### DISCUSSION TITLE
some tailwind classes not working properly on next js 13?

### DISCUSSION THREAD

**Discussion by clxdia** [tailwindlabs/tailwindcss#10048] [Help] [answered]
Hello, i'm currently learning nextjs version 13 (the experimental App folder specifically) and i was moving some components around in another folder and all of a sudden the ui on my project started breaking up on some of them only? For my header, for example, the height and display flex properties don't work anymore but the text-color and text size do. No errors appear on the console, does anyone know what's causing this i'm losing my mind

**Comment by wongjn** [comment, marked as answer]
Perhaps check that the `content` file globs in the Tailwind configuration sufficiently cover your source code.
